pilot/pkg/model/telemetry.go

```diff
@@ -1086,6 +1094,22 @@ var metricToPrometheusMetric = map[string]string{
 	"GRPC_RESPONSE_MESSAGES": "response_messages_total",
 }
 
+func metricRotationInterval() *durationpb.Duration {
```

Comment:
nit: you can make features.MetricRotationInterval a durationpb.Duration by computing it once when you assign it. I think we do it with some others

Reply:
durationpb.Duration does not satisfy comparable